### PR TITLE
Checks that text appears before another text.

### PR DESCRIPTION
**Pull request by Eduardo-Morales-Alberti**
Create step to check that text appears before another text.

### CODE REVIEW

File: Behat/Context/DrupalExtendedContext.php [hunk 1]
```diff
@@ -497,4 +497,32 @@ public function userWithMailNotExists($mail) {
      $this->userWithMailExists($mail, FALSE);
    }
 
+  /**
+   * Checks that text appears before another text.
+   *
+   * Example:
+   * I should see "Element 1" text before "Element 2" from "li" element.
```
Comment (rsanzante) [inline Behat/Context/DrupalExtendedContext.php]: As agreed by chat let's use a region instead of and HTML element to limit the scope.

Comment (Eduardomoberti) [reply Behat/Context/DrupalExtendedContext.php]: Example region: https://github.com/jhedstrom/drupalextension/blob/c4cee779960576830628fddb87cfebc7649166a9/src/Drupal/DrupalExtension/Context/MinkContext.php#L28

**Comment by Eduardomoberti** [comment]
@rsanzante I already added the region.

File: Behat/Context/DrupalExtendedContext.php [hunk 2]
```diff
@@ -497,4 +497,38 @@ public function userWithMailNotExists($mail) {
...
```
Comment (rsanzante) [inline Behat/Context/DrupalExtendedContext.php]: «I should see "Element 1" text before "Element 2" from "li" element.» -> «I should see "Element 1" text before "Element 2" in the "content" region»

**Pull request merged**